src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-list/rbd-list.component.html

```diff
@@ -41,3 +42,19 @@
   <strong>{{ value.parent }}</strong> to child
   <strong>{{ value.child }}</strong>.
 </ng-template>
+
+<ng-template #deleteTpl
+             let-hasSnapshots="hasSnapshots"
+             let-snapshots="snapshots">
+  <div class="alert alert-warning"
+       *ngIf="hasSnapshots"
+       role="alert">
+    <span i18n>Deleting this image will also delete all its snapshots.</span>
+    <br>
+    <ng-container *ngIf="snapshots.length > 0">
+      <span i18n>The following snapshots are currently protected and will be removed:</span>
+      <br>
+      {{ snapshots | join }}
```

Comment:
I would prefer a listing here instead of a comma separated list.

Currently:
clone-rbd1_2020-03-06T09:22:48.645+01:00, clone-rbd1_2020-03-06T09:22:52.505+01:00

Wanted:
* clone-rbd1_2020-03-06T09:22:48.645+01:00
* clone-rbd1_2020-03-06T09:22:52.505+01:00

Reply:
Done.
![image](https://user-images.githubusercontent.com/399326/76090806-c1947600-5fab-11ea-91d3-0922fc27d073.png)

Reply:
Just curious: how does this form deal with hundreds of snapshots? This is not an uncommon scenario.

Reply:
Only protected snapshots will be shown @LenzGr .

Reply:
How about this? I set up a max height of around 5 rows. It will not fill the whole screen and the user can still check every protected snapshot that will be removed.
![image](https://user-images.githubusercontent.com/399326/76226522-7d081500-620d-11ea-86d4-0502bf934fa7.png)